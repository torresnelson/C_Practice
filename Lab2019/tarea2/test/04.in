# prueba de insertar_segmento_despues, copiar_segmento, cortar_segmento, precede_en_cadena, menor_en_cadena, siguiente_clave
precede_en_cadena 1
insertar_segmento_despues .
es_vacia_cadena
imprimir_cadena
precede_en_cadena 1
copiar_segmento 1 1
cortar_segmento 1 1
insertar_segmento_despues (1,a) .
es_vacia_cadena
imprimir_cadena
copiar_segmento 1 1
cortar_segmento 1 1
insertar_segmento_despues (2,b) (3,c) (4,d) (5,e) .
imprimir_cadena
inicio_cadena
precede_en_cadena 1
precede_en_cadena 2
siguiente
insertar_segmento_despues .
imprimir_cadena
insertar_segmento_despues (6,f) .
imprimir_cadena
final_cadena
precede_en_cadena 1
precede_en_cadena 5
insertar_segmento_despues (7,g) (8,h) .
imprimir_cadena
insertar_segmento_despues (9,i) (10,j) .
precede_en_cadena 4
precede_en_cadena 5
precede_en_cadena 6
imprimir_cadena
copiar_segmento 1 1
copiar_segmento 5 5
copiar_segmento 9 9
copiar_segmento 1 9
copiar_segmento 2 8
cortar_segmento 9 9
cortar_segmento 1 1
cortar_segmento 4 4
cortar_segmento 4 6
cortar_segmento 1 2
cortar_segmento 1 1
es_vacia_cadena
insertar_segmento_despues (2,b) (3,c) (4,d) (1,a) .
inicio_cadena
info_cadena
menor_en_cadena
info_cadena
insertar_segmento_despues (5,e) (6,f) (7,g) (0,0) (8,h) .
menor_en_cadena
info_cadena
siguiente_clave 1
es_localizador
final_cadena
menor_en_cadena
info_cadena
siguiente_clave 8
es_localizador
info_cadena
kesimo 2
siguiente_clave 3
info_cadena
siguiente_clave 7
info_cadena
Fin