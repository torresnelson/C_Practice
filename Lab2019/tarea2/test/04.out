1>#  prueba de insertar_segmento_despues, copiar_segmento, cortar_segmento, precede_en_cadena, menor_en_cadena, siguiente_clave.
2>Posición fuera de rango.
3>Segmento insertado después de loc.
4>cad vacia.
5>
6>Posición fuera de rango.
7>
8>
9>Segmento insertado después de loc.
10>cad no vacia.
11>(1,a)
12>(1,a)
13>
14>Segmento insertado después de loc.
15>(2,b)(3,c)(4,d)(5,e)
16>loc al inicio.
17>loc1 precede a loc.
18>loc1 no precede a loc.
19>loc al siguiente.
20>Segmento insertado después de loc.
21>(2,b)(3,c)(4,d)(5,e)
22>Segmento insertado después de loc.
23>(2,b)(3,c)(6,f)(4,d)(5,e)
24>loc al final.
25>loc1 precede a loc.
26>loc1 precede a loc.
27>Segmento insertado después de loc.
28>(2,b)(3,c)(6,f)(4,d)(5,e)(7,g)(8,h)
29>Segmento insertado después de loc.
30>loc1 precede a loc.
31>loc1 precede a loc.
32>loc1 no precede a loc.
33>(2,b)(3,c)(6,f)(4,d)(5,e)(9,i)(10,j)(7,g)(8,h)
34>(2,b)
35>(5,e)
36>(8,h)
37>(2,b)(3,c)(6,f)(4,d)(5,e)(9,i)(10,j)(7,g)(8,h)
38>(3,c)(6,f)(4,d)(5,e)(9,i)(10,j)(7,g)
39>(2,b)(3,c)(6,f)(4,d)(5,e)(9,i)(10,j)(7,g)
40>(3,c)(6,f)(4,d)(5,e)(9,i)(10,j)(7,g)
41>(3,c)(6,f)(4,d)(9,i)(10,j)(7,g)
42>(3,c)(6,f)(4,d)
43>(4,d)
44>
45>cad vacia.
46>Segmento insertado después de loc.
47>loc al inicio.
48>(2,b)
49>loc avanzó hasta el menor.
50>(1,a)
51>Segmento insertado después de loc.
52>loc avanzó hasta el menor.
53>(0,0)
54>loc avanzó buscando 1.
55>loc no válido.
56>loc al final.
57>loc avanzó hasta el menor.
58>(8,h)
59>loc avanzó buscando 8.
60>loc válido.
61>(8,h)
62>loc en la posición 2.
63>loc avanzó buscando 3.
64>(3,c)
65>loc avanzó buscando 7.
66>(7,g)
67>Fin.
